sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs

```diff
@@ -102,7 +102,7 @@ public async Task<long> GetMessageCountAsync()
             }
             catch (Exception e)
             {
-                _logger.LogWarning($"Error querying for Service Bus {entityName} scale status: {e.Message}");
+                _logger.LogWarning($"Error querying for Service Bus {entityName} scale status", e);
```

Comment:
Is this change intended?

Reply:
this is outdated, I fixed this by moving the exception to the first parameter.